File: 241122/바이러스 검사/virus-detector.cpp
#include <iostream>
using namespace std;

int main() {
    int N, i, lead, mate;
    cin >> N;
    int* nums = new int[N];
    for(i=0;i<N;i++) cin >> nums[i];
    cin >> lead >> mate;
    int answer = N;
    for(i=0;i<N;i++) {
        if (nums[i] <= lead) continue;
        nums[i] -= lead;
        answer += (nums[i]/mate);
        if (nums[i]%mate == 0) answer++; 
    }
    cout << answer;
    return 0;
}

File: 241122/바이러스 검사/virus-detector.py
import sys
input = sys.stdin.readline
from math import ceil

N = int(input())
customers = list(map(int, input().split()))
lead, mate = map(int, input().split())

answer = len(customers)

for num in customers:
    if num <= lead:
        continue
    num -= lead
    answer += ceil(num / mate)
print(answer)